examples/99-advanced/atomic-orbitals.py

```diff
@@ -0,0 +1,240 @@
+)
+contours = contours.interpolate(grid)
+contours.plot(
+    smooth_shading=True,
+    show_scalar_bar=False,
+)
+
+
+###############################################################################
+# Volumetric Plot: Plot the Orbitals using RGBA
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Let's now combine some of the best parts of the two above plots. The
+# volumetric plot is great for showing the probability of the "electron cloud"
+# orbitals, but the colormap doesn't quite match reality as well as the
+# isosurface plot.
+#
+# For this example we're going to use an RGBA colormap to tightly control the
+# way the orbitals are plotted. For this, the opacity will be mapped to the
+# probability of the electron being at a location in the grid, which we can do
+# by taking the absolute value squared of the orbital's wave function. We can
+# set the color of the orbital based on the phase, which we can get simply by
+# with ``orbital['real_wf'] < 0``.
+#
+# Let's start with a simple one, the :math:`3p_z` orbital.
+
+
+def plot_orbital(orbital, cpos='iso', clip_plane='x'):
+    """Plot an electron orbital using an RGBA colormap."""
+    neg_mask = orbital['real_wf'] < 0
+    rgba = np.zeros((orbital.n_points, 4), np.uint8)
+    rgba[neg_mask, 0] = 255
+    rgba[~neg_mask, 1] = 255
+
+    # normalize opacity
+    opac = np.abs(orbital['real_wf']) ** 2
+    opac /= opac.max()
+    rgba[:, -1] = opac * 255
+
+    orbital['plot_scalars'] = rgba
+
+    pl = pv.Plotter()
+    vol = pl.add_volume(
+        orbital,
+        scalars='plot_scalars',
+    )
+    vol.prop.interpolation_type = 'linear'
+    if clip_plane:
+        pl.add_volume_clip_plane(
+            vol,
+            normal=clip_plane,
+            normal_rotation=False,
+        )
+    pl.camera_position = cpos
+    pl.camera.zoom(1.5)
+    pl.show_axes()
+    return pl.show()
+
+
+hydro_orbital = examples.load_hydrogen_orbital(3, 1, 0)
+plot_orbital(hydro_orbital, clip_plane='-x')
+
+
+###############################################################################
+# Volumetric Plot: 4dz2 orbital
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+hydro_orbital = examples.load_hydrogen_orbital(4, 2, 0)
+plot_orbital(hydro_orbital, clip_plane='-y')
+
+
+###############################################################################
+# Volumetric Plot: 4dxz orbital
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+hydro_orbital = examples.load_hydrogen_orbital(4, 2, -1)
+plot_orbital(hydro_orbital, clip_plane='-y')
+
+
+###############################################################################
+# Plot an Orbital Using a Density Plot
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We can also plot atomic orbitals using a 3D density plot. For this, we will
+# use :func:`numpy.random.choice` to sample all the points of our
+# :class:`pyvista.UniformGrid` based on the probability of the electron being
+# at that coordinate.
+#
+# Note how we square the probability to better visualize the distribution of
```

Comment:
I don't think we square the probability: looks like we're squaring the wave function, which is itself the probability (density). This sentence might be unnecessary anyway considering the "absolute value squared" explanation in the previous section.

Reply:
Fixed in [7417ab3](https://github.com/pyvista/pyvista/pull/3825/commits/7417ab3e4ea2f340133b3e3716d08cfc99ff543e)